eng/common/scripts/Update-DocsMsPackages.ps1

```diff
@@ -61,6 +61,7 @@ function GetDocsMetadataForMoniker($moniker) {
       RepoPath = $fileObject.ServiceDirectory;
       Type = $fileObject.SdkType;
       New = $fileObject.IsNewSdk;
+      GroupId = $fileObject.Gruop;
```

Comment:
```suggestion
      GroupId = $fileObject.Group;
```

Reply:
Make changes to add member once we get the property